tools/dgeni/templates/class.template.html

```diff
@@ -3,14 +3,21 @@ <h4 class="docs-api-h4 docs-api-class-name">
 </h4>
 <p class="docs-api-class-description">{$ class.description $}</p>
 
+{%- if class.directiveSelectors -%}
+<span class="docs-api-h4 docs-api-class-selector-label">Selectors:</span>
```

Comment:
Can you wrap the inside of this if block in a `<div class="docs-api-directive-selectors>"?